qiskit/execute.py

```diff
@@ -131,3 +133,79 @@ def execute_circuits(circuits, backend, qobj_header=None,
 
     # executing the circuits on the backend and returning the job
     return backend.run(qobj, **kwargs)
+
+
+def execute_schedules(schedules, backend, user_lo_dicts=None, **kwargs):
+    """Executes a list of circuits.
+
+    Args:
+        schedules (Schedule or list[Schedule]): schedules to execute
+        backend (BaseBackend): a backend to execute the schedules on
+        user_lo_dicts (UserLoDict or list[UserLoDict]): Dictionaries of user LO frequencies
+        kwargs: extra arguments to configure backend
+
+    Kwargs:
+        shots (int): number of repetitions of each circuit, for sampling
+        max_credits (int): maximum credits to use
+        seed (int): random seed for simulators
+        meas_level (int): set the appropriate level of the measurement output.
+        memory_slots (int): number of classical memory slots used in this job.
+        memory_slot_size (int): size of each memory slot if the output is Level 0.
+        meas_return (str): indicates the level of measurement information to return.
+            "single" returns information from every shot of the experiment.
+            "avg" returns the average measurement output (averaged over the number of shots).
+            If the meas level is 2 then this is fixed to single.
+        rep_time (int): repetition time of the experiment in μs.
+            The delay between experiments will be rep_time.
+            Must be from the list provided by the device.
+
+    Returns:
+        BaseJob: returns job instance derived from BaseJob
+
+    Raises:
+        PulseError: when #schedules : #user_lo_dicts is not either of 1:n, n:1 or n:n.
+    """
+    if isinstance(schedules, Schedule):
+        schedules = [schedules]
+
+    if user_lo_dicts:
+        if isinstance(user_lo_dicts, UserLoDict):
+            user_lo_dicts = [user_lo_dicts]
+        if len(schedules) == 1:
+            experiments = [ConditionedSchedule(schedules[0], cond) for cond in user_lo_dicts]
+        elif len(user_lo_dicts) == 1:
```

Comment:
If only one `lo_config` is provided, we need to update `PulseQobj.PulseQobjConfig` rather than `PulseQobj.PulseQobjExperiment.PulseQobjExperimentConfg`.